apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/ConnectionPoolOptions.cs

```diff
@@ -146,6 +123,10 @@ public ResourcesExhaustedBehavior ResourcesExhaustedBehavior
 
         /// <summary>
         /// </summary>
-        public int TimeoutMilliseconds { get; set; } = 600000;
+        public TimeSpan Timeout
```

Comment:
Thanks for the change - I assume the docs will make it clear what *kind* of timeout this is.